src/packages.xml

```diff
@@ -0,0 +1,216 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file used for anything? It seems like it's a remnant.

Reply:
This file is indeed a remnant. Resolved here https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/3e0a071db2e5ea2095623e3b6a4af251c1893a13